libraries/AP_HAL_ChibiOS/hwdef/SkystarsF405DJI/hwdef.dat

```diff
@@ -0,0 +1,148 @@
+# Skystars F405 DJI
+# With F405 MCU, MPU6000 IMU and 7456 series OSD
+# Based on Mamba F405 MK2 from jeanphilippehell
+# Pin information from Betaflight
+
+# MCU class and specific type.
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+# copied from the same board, must be changed in the future
+# to avoid conflicts
+APJ_BOARD_ID 1019
```

Comment:
Please reserve a new  board ID now - and put it here.

What's the ETA on the board?

Reply:
Hello Peter!
Thanks for the support.
Reserved ID 146, next on [this list](https://github.com/ArduPilot/ardupilot/blob/master/Tools/AP_Bootloader/board_types.txt).
I'm new to this so I'm not sure how to include this number so bootloader can recognise it as a valid ID.
Please, let me know the next steps.
About the ETA, the board is already in the market. I just ported Ardupilot to it since it wasn't available yet.

Reply:
It's important to mention I couldn't make Smart Audio work yet (just in Betaflight with same connections and hardware). My partner asked for help [here](https://www.facebook.com/groups/ArduPilot.org/posts/4280995325256961/).